static/app/components/feedback/hydrateFeedbackRecord.tsx

```diff
@@ -5,6 +5,7 @@ export default function hydrateFeedbackRecord(
 ): HydratedFeedbackItem {
   return {
     ...data,
+    feedback_id: data.feedback_id.replaceAll('-', ''),
```

Comment:
I think I saw a slack post somewhere about this... +1 to cleaning this on backend?

Reply:
https://github.com/getsentry/sentry/pull/55566/files#r1326537633